packages/server-wallet/src/wallet/wallet.ts

```diff
@@ -200,6 +203,13 @@ export class SingleThreadedWallet extends EventEmitter<EventEmitterType>
 
     response.queueChannelRequest(channelId, myIndex, participants);
     response.queueChannelState(channelState);
+
+    if (await this.store.isLedger(channelId)) {
```

Comment:
I don't think I have the contextual awareness to actually review this component anymore 😞